docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -65,6 +68,18 @@ as any order. For example, you can write COMMENT table_comment after TBLPROPERTI
 
     Partitions are created on the table, based on the columns specified.
     
+* **CLUSTERED BY**
+
+    Specifies bucket columns for bucketing table.
+    
+* **SORTED BY**
+
+    Used to sort bucket column, we can combine with `ASC` for ascending order, with `DESC` for descending order.
```

Comment:
How about rephrasing it like this? `Specifies an ordering of bucket columns. Optionally, one can use ASC for an ascending order or DESC  for a descending order after any column names in the SORTED BY clause. If not specified, ASC is assumed by default.`